tests/e2e/machine_pool_test.go

```diff
@@ -483,3 +487,276 @@ var _ = Describe("TF Test", func() {
+					Expect(mpResponseBody.Autoscaling().MaxReplicas()).To(Equal(maxReplicas))
+				} else {
+					By("Edit default machinepool with replicas")
+					replicas := 6
+					dmpArgFromMachinepoolForTesting.Replicas = &replicas
+					_, err := dmpService.Create(&dmpArgFromMachinepoolForTesting)
+					Expect(err).ToNot(HaveOccurred())
+
+					By("Verify the parameters of the created machinepool")
+					mpResponseBody, err := cms.RetrieveClusterMachinePool(ci.RHCSConnection, clusterID, defaultMPName)
+					Expect(err).ToNot(HaveOccurred())
+					Expect(mpResponseBody.Replicas()).To(Equal(replicas))
+				}
+
+				By("Edit default machinepool with labels")
+				dmpArgFromMachinepoolForTesting = defaultMachinepoolArgFromMachinepool
+				creationLabels := map[string]string{"fo1": "bar1", "fo2": "baz2"}
+				dmpArgFromMachinepoolForTesting.Labels = creationLabels
+				_, err = dmpService.Create(&dmpArgFromMachinepoolForTesting)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Verify the parameters of the created machinepool")
+				mpResponseBody, err := cms.RetrieveClusterMachinePool(ci.RHCSConnection, clusterID, defaultMPName)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(mpResponseBody.Labels()).To(Equal(creationLabels))
+
+				By("Create an additional machinepool")
+				replicas := 3
+				machineType := "r5.xlarge"
+				name := "amp-69009"
+
+				MachinePoolArgs := &exe.MachinePoolArgs{
+					Token:       token,
+					Cluster:     clusterID,
+					Replicas:    &replicas,
+					MachineType: machineType,
+					Name:        name,
+					Taints:      taints,
+				}
+
+				_, err = mpService.Create(MachinePoolArgs)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Edit the default machinepool with taints")
+				dmpArgFromMachinepoolForTesting = defaultMachinepoolArgFromMachinepool
+				_, err = dmpService.Create(&defaultMachinepoolArgFromMachinepool)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Verify the parameters of the created machinepool")
+				mpResponseBody, err = cms.RetrieveClusterMachinePool(ci.RHCSConnection, clusterID, name)
+				Expect(err).ToNot(HaveOccurred())
+				respTaints := mpResponseBody.Taints()
+				for index, taint := range respTaints {
+					Expect(taint.Effect()).To(Equal(taints[index]["schedule_type"]))
+					Expect(taint.Key()).To(Equal(taints[index]["key"]))
+					Expect(taint.Value()).To(Equal(taints[index]["value"]))
+				}
+			})
+		})
+
+	})
+})
+
+// var _ = Describe("TF Test, day-0 default machinepool testing", func() {
+// 	Describe("Default MachinePool day-0 test cases", func() {
+
+// 		var (
+// 			dmpService             *exe.MachinePoolService
+// 			defaultMachinePoolArgs exe.MachinePoolArgs
+// 			mpService              *exe.MachinePoolService
+// 		)
+
+// 		BeforeEach(func() {
+// 			dmpService = exe.NewMachinePoolService(con.DefaultMachinePoolDir)
+// 			mpService = exe.NewMachinePoolService(con.MachinePoolDir)
+
+// 			By("Make sure the default machinepool imported from cluster state")
```

Comment:
if it is not used - can it be removed?

Reply:
I will confirm with @xueli181114 if there is some job config which can be used as condition to skip this test cases. Thanks.